gsa/src/web/pages/reports/details/emptyreport.js

```diff
@@ -78,7 +78,7 @@ const EmptyReport = ({
             {_('Just wait for results to arrive.')}
           </ReportPanel>
         )}
-        {progress < 1 && hasTarget && (
+        {!isActiveReport && hasTarget && (
```

Comment:
I think we have to keep the "progress < 1" too. I would assume it covers the case where a scan is already running for a while but there isn't any progress yet because the scanned hosts are dead.

Reply:
I have added the dead host note to conditions `progress < 1` and `!isActiveReport`